libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -427,7 +403,9 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
         FragmentActivity activity = getActivity();
 
         Bundle arguments = getArguments();
-        boolean supportStockPhotos = arguments != null && arguments.getBoolean(ARG_SITE_USING_WPCOM_REST_API);
+        GutenbergWebViewAuthorizationData gutenbergWebViewAuthorizationData =
+                arguments.getParcelable(ARG_GUTENBERG_WEB_VIEW_AUTH_DATA);
```

Comment:
Should we check for `arguments` being `null` here? The next two lines that use the arguments do that with `arguments != null && ...`. Obviously we can't do that here since it's not a boolean, but I think we could achieve that effect if we inline this variable where it is used on line 409, since that is a boolean and has the null check.

There are actually a few places where we don't null check `getArguments()` in this class. I actually don't feel strongly about null-checking `getArguments()` because they should never be null as long as they are set (which we do), _but_ I do think we should be consistent--either we always check them or we never do. Having a mix and _sometimes_ checking them seems confusing. We seem closer to always checking them, and that's obviously a bit safer, so that would be my preference but, like I said, I don't feel particularly strongly (particularly if there is no reasonable way to handle a particular case where `getArguments()` is null). WDYT?

Reply:
Thanks, @mchowning! Good point here. We should pick a direction and probably we are closer to check `getArguments()` to not be null.

Fixed.